pkg/corpus/corpus.go

```diff
@@ -88,6 +106,29 @@ type NewItemEvent struct {
 	NewCover []uint64
 }
 
+// SetFocusAreas can only be called once on an empty corpus.
+func (corpus *Corpus) SetFocusAreas(areas []FocusArea) {
```

Comment:
Is the only reason to set it lazily `mgr.http` object where we pass corpus during constructions?
If so, I would move the burden to `mgr.http`. It already has a number of atomic.Pointer's that are set when machine is checked. It would be reasonable to create the corpus when machine is checked.